scripts/shared/nodeinfo/nodeinfo.json

```diff
@@ -7,15 +7,15 @@
         "uri": "https://tomesh.net"
     },
     "software": {
-        "repo": "REPO",
-        "branch": "BRANCH",
-        "commit": "COMMIT",
-        "installed": "INSTALLED",
+        "repo": "__REPO__",
+        "branch": "__BRANCH__",
+        "commit": "__COMMIT__",
+        "installed": "__INSTALLED__",
         "uri": "https://github.com/tomeshnet/prototype-cjdns-pi/"
     },
-    "key": "KEY",
-    "org": "ORG",
+    "key": "__KEY__",
+    "org": "__ORG__",
     "services": {
-        SERVICES
+        __SERVICES__
```

Comment:
It's better to have stable keys for validation and navigation purposes. Like:

```
"services": [
    {
        "name": "ipfs",
        "version": "__IPFS_VERSION__",
        "params": [__IPFS_PARAMS__]
    }
]
```

This is a very rushed schema design. My take is it will completely change in the future, so stick a version on the schema.

Reply:
Also rolled into issue #173

Reply:
@benhylau That is what the original nodeinfo spec said to do. Would you rather this be changed before the release? And where should the version be recorded?

Reply:
The only thing we need to change in this release is add:

`"version": 0,`

to the top level, as a sibling to `"services"`. This will allow us to detect breaking changes and since this is prototype, we will break backwards compatibility as needed :)

My point is, schema design is something that will take a long time to discuss, and even then it will evolve. Adding a version is absolutely a smart thing to do now.

Reply:
Added in https://github.com/tomeshnet/prototype-cjdns-pi/commit/3e8e30fe2dadbb72362d43d59a45bdb3c58b44d2